CMake/RealmCore.cmake

```diff
@@ -113,6 +113,15 @@ function(download_realm_core core_version)
     set_property(TARGET realm PROPERTY IMPORTED_LOCATION_RELEASE ${core_library_release})
     set_property(TARGET realm PROPERTY IMPORTED_LOCATION ${core_library_release})
 
+    if (LINUX)
+        find_package(Threads)
+        find_package(PkgConfig REQUIRED)
+        pkg_search_module(OPENSSL REQUIRED openssl)
```

Comment:
The two aren't equivalent AFAIK, but `find_package(OpenSSL REQUIRED)` seems like it should be sufficient. Is there a reason `pkg_search_module` was used instead?